tests/toolkit/timeseries_tests.cc

```diff
@@ -0,0 +1,27 @@
+#include <gtest/gtest.h>
+
+#include "agent.h"
+
+#include "../test_context.h"
+#include "../test_agents/test_agent.h"
+
+namespace cyclus {
+namespace toolkit {
+
+TEST(TimeSeriesTests, Power) {
+  TestContext tc;
+  Agent* a = new TestAgent(tc.get());
+  RecordTimeSeries<POWER>(a, 42.0);
+  delete a;
+}
+
+TEST(TimeSeriesTests, RawPower) {
+  TestContext tc;
+  Agent* a = new TestAgent(tc.get());
+  RecordTimeSeries<double>("Power", a, 42.0);
+  delete a;
+}
```

Comment:
No, this doesn't write out to a real database. It just implements the correct interface for testing.  This strategy is used in other parts of the test suite. We test database writing in other places as well.